src/app.ts

```diff
@@ -167,9 +169,7 @@ async function app() {
     }
     config = readConfig(originalInput);
     tank = fillTank(rawFSH, config);
-  } catch (e) {
-    logger.error('SUSHI encountered an error while processing FSH and configuration files: ', e);
```

Comment:
Do we really want to remove this error message too?  If a message makes it to this catch block, isn't there a good chance that it was never logged?  And now it will fail silently?

I think it makes sense not to output the help, but maybe we should still log that catch-all error message just in case.